src/crypto/crypto_context.cc

```diff
@@ -854,6 +854,16 @@ void SecureContext::SetDHParam(const FunctionCallbackInfo<Value>& args) {
 
   CHECK_GE(args.Length(), 1);  // DH argument is mandatory
 
+  // If the user specified "auto" for dhparams, the JavaScript layer will pass
+  // true to this function instead of the original string. Any other string
+  // value will be interpreted as custom DH parameters below.
+  if (args[0]->IsTrue()) {
+    if (!SSL_CTX_set_dh_auto(sc->ctx_.get(), true)) {
```

Comment:
This can be a CHECK because it's not supposed to fail. Something is very wrong if it does.

Reply:
Updated, thanks. PTAL.